plugins/rbac-backend/src/plugin.ts

```diff
@@ -54,7 +53,7 @@ export const rbacPlugin = createBackendPlugin({
         auth,
         httpAuth,
       }) {
-        const winstonLogger = loggerToWinstonLogger(logger);
+        const winstonLogger = logger;
 
```

Comment:
```suggestion
```

Sadly it's not allowing me to make the full suggestion. Do the same thing as above and remove `winstonLogger`.